test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -415,6 +415,7 @@ describe("UserWallet - Lightning Pay", () => {
         // https://github.com/alexbosworth/ln-service/issues/122
         await settleHodlInvoice({ lnd: lndOutside1, secret })
 
+        await sleep(5000)
```

Comment:
Is there a way we can poll when we can continue instead of sleep 5 seconds? This number seems arbitrary and is probably not optimal for everyone.

Reply:
Can also leave this for a future pass.

Reply:
was in the original test, I tried to remove syncing nodes, waiting for pending balance and subscribing to update invoice event... seems to be the cancel event and the sync issues between lnds

this is the related issue https://github.com/GaloyMoney/galoy/pull/340/commits/1e36a02847c49efbad52ba0c8ba738e94ddef20b

Reply:
let me try one last thing before giving up on this... (also maybe is not needed in your environment, only in mine)

Reply:
ok, a bit messy but I think is finally fixed https://github.com/GaloyMoney/galoy/pull/340/commits/7401863bcf37db1d293eeb5269d892afd353224a